charts/fybrik/files/taxonomy/datacatalog.json

```diff
@@ -0,0 +1,113 @@
+{
+  "title": "datacatalog.json",
+  "definitions": {
+    "GetAssetRequest": {
+      "type": "object",
+      "required": [
+        "assetID",
+        "operationType"
+      ],
+      "properties": {
+        "assetID": {
+          "$ref": "taxonomy.json#/definitions/AssetID"
+        },
+        "operationType": {
+          "$ref": "#/definitions/OperationType"
+        }
+      }
+    },
+    "GetAssetResponse": {
+      "type": "object",
+      "required": [
+        "resourceMetadata",
+        "details",
+        "credentials"
+      ],
+      "properties": {
+        "credentials": {
+          "description": "This has the vault plugin path where the data credentials will be stored as kubernetes secrets This value is assumed to be known to the catalog connector.",
+          "type": "string"
+        },
+        "details": {
+          "$ref": "#/definitions/ResourceDetails"
+        },
+        "resourceMetadata": {
+          "$ref": "#/definitions/ResourceMetadata"
+        }
+      }
+    },
+    "OperationType": {
+      "description": "OperationType Type of operation requested for the asset",
+      "type": "string",
+      "enum": [
+        "read"
```

Comment:
Perhaps to add also write and delete for future uses?

Reply:
Keeping it as is for now because that's the only value in the existing taxonomy, but before closing the issue a full final review of the taxonomy definitions is required.